modules/EvseManager/evse/evse_managerImpl.cpp

```diff
@@ -168,12 +168,16 @@ void evse_managerImpl::ready() {
 
             se.session_started = session_started;
         } else if (e == types::evse_manager::SessionEventEnum::SessionFinished) {
+            types::evse_manager::SessionFinished session_finished;
+            session_finished.timestamp =
+                date::format("%FT%TZ", std::chrono::time_point_cast<std::chrono::milliseconds>(date::utc_clock::now()));
```

Comment:
Please use the following to format timestamps in everest:

Everest::Date::to_rfc3339(date::utc_clock::now());

Reply:
Done. I wildly copy-pasted what is done for `SessionStarted` events. I might as well create a different MR to use the format you specified as well in `SessionStarted`, `TransactionStarted` and `TransactionFinished` timestamps' events and unify them.

Reply:
That sounds like a very good idea!